docs/monitoring.md

```diff
@@ -224,10 +224,12 @@ both running applications, and in the history server.  The endpoints are mounted
 for the history server, they would typically be accessible at `http://<server-url>:18080/api/v1`, and
 for a running application, at `http://localhost:4040/api/v1`.
 
-In the API, an application is referenced by its application ID, `[app-id]`.
-When running on YARN, each application may have multiple attempts; each identified by their `[attempt-id]`.
-In the API listed below, `[app-id]` will actually be `[base-app-id]/[attempt-id]`,
-where `[base-app-id]` is the YARN application ID.
+In the API, an application is referenced by its application ID, `[app-id]`.
+Spark on YARN supports multiple application attempts in cluster mode but not in client mode.
```

Comment:
yarn client mode also supports multiple attempts, the description is not exact. Actually what you want to deliver is that attempId is not required for yarn client mode, would you please consolidate your words.